packages/cache-browser-local-storage/src/createBrowserLocalStorageCache.ts

```diff
@@ -26,10 +26,13 @@ export function createBrowserLocalStorageCache(options: BrowserLocalStorageOptio
       events: CacheEvents<TValue> = {
         miss: () => Promise.resolve(),
```

Comment:
these parameters need to stay because we are using the generic in getNameSpace. See https://twitter.com/haroenv/status/1252610543162179585